app/forms/waste_exemptions_engine/renewal_start_form.rb

```diff
@@ -4,6 +4,8 @@ module WasteExemptionsEngine
   class RenewalStartForm < BaseForm
     include DataOverviewForm
```

Comment:
I think we can get rid of this concern now as the DataOverviewPresenter has basically replaced it

Reply:
Have removed the mixin. The `DataOverviewForm` module is still used by `edit_form`, so have left it. Does that sound right?

Reply:
Oops yeah, forgot about that one! We can ditch it soon then